src/server/game/Globals/ObjectMgr.h

```diff
@@ -1263,7 +1263,15 @@ class ObjectMgr
     }
     [[nodiscard]] QuestGreetingLocale const* GetQuestGreetingLocale(TypeID type, uint32 id) const
     {
-        QuestGreetingLocaleContainer::const_iterator itr = _questGreetingLocaleStore.find(MAKE_PAIR32(type, id));
+        uint32 typeIndex;
+        if (type == TYPEID_UNIT)
```

Comment:
[Please include brackets with the if statement](https://www.youtube.com/watch?v=8RrQrATnXXY&themeRefresh=1). ❤️